src/components/postBoard.tsx

```diff
@@ -5,14 +5,40 @@ import Post from './post';
 
 class PostBoard extends Component {
 
+    state = {
+        userPosts: [
+            {
+                userId: 0,
+                id: 0,
+                title: "None",
+                completed: false
+            }
+        ]
+    };
+
+    componentDidMount(){
+     
+        fetch('https://jsonplaceholder.typicode.com/posts')
+        .then(response => response.json())
+        .then(json => {
+            let userPosts = json;
+            this.setState({userPosts});
+        });
+      
+    }
+
     render(){
 
         return(
             <React.StrictMode>
-                <Post/>
-                <Post/>
-                <Post/>
-                <Post/>
+                {this.state.userPosts.map(post => (
```

Comment:
This looks good. You could even implement a search function here that filters this array before mapping it